app/controllers/schools/year2020_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 module Schools
   class Year2020Controller < ApplicationController
+    before_action :set_headers
```

Comment:
Could we call this `set_no_index_headers`? I can imagine a `set_headers` being defined on ApplicationController